src/common/verify/isValidAccessGrant.test.ts

```diff
@@ -190,6 +192,8 @@ describe("isValidAccessGrant", () => {
       .mocked(getVerifiableCredentialApiConfiguration)
       .mockResolvedValueOnce({
         verifierService: "https://some.vc.verifier",
+        specCompliant: {},
+        legacy: {},
```

Comment:
Oh I see, the mocks return type wasn't compliant with the new type, was this the issue?

Reply:
Yup